### ISSUE TITLE
tests for secondary indexes

### ISSUE THREAD

**Issue by mick** [mapbox/dyno#26]

**Comment by sbma44** [comment]
related, export/import appears to fail when a secondary index is present:

```
dyno us-east-1 export [hourly-table] | head -n 5000 | dyno us-east-1 import [currently-nonexistent-table]
{ [MultipleValidationErrors: There were 4 validation errors:
* UnexpectedParameter: Unexpected key 'LastDecreaseDateTime' found in params.GlobalSecondaryIndexes[0].ProvisionedThroughput
* UnexpectedParameter: Unexpected key 'LastIncreaseDateTime' found in params.GlobalSecondaryIndexes[0].ProvisionedThroughput
* UnexpectedParameter: Unexpected key 'LastDecreaseDateTime' found in params.ProvisionedThroughput
* UnexpectedParameter: Unexpected key 'LastIncreaseDateTime' found in params.ProvisionedThroughput]
  message: 'There were 4 validation errors:\n* UnexpectedParameter: Unexpected key \'LastDecreaseDateTime\' found in params.GlobalSecondaryIndexes[0].ProvisionedThroughput\n* UnexpectedParameter: Unexpected key \'LastIncreaseDateTime\' found in params.GlobalSecondaryIndexes[0].ProvisionedThroughput\n* UnexpectedParameter: Unexpected key \'LastDecreaseDateTime\' found in params.ProvisionedThroughput\n* UnexpectedParameter: Unexpected key \'LastIncreaseDateTime\' found in params.ProvisionedThroughput',
  code: 'MultipleValidationErrors',
  errors: 
   [ { [UnexpectedParameter: Unexpected key 'LastDecreaseDateTime' found in params.GlobalSecondaryIndexes[0].ProvisionedThroughput]
       message: 'Unexpected key \'LastDecreaseDateTime\' found in params.GlobalSecondaryIndexes[0].ProvisionedThroughput',
       code: 'UnexpectedParameter',
       time: Thu Feb 05 2015 16:30:05 GMT-0800 (PST) },
     { [UnexpectedParameter: Unexpected key 'LastIncreaseDateTime' found in params.GlobalSecondaryIndexes[0].ProvisionedThroughput]
       message: 'Unexpected key \'LastIncreaseDateTime\' found in params.GlobalSecondaryIndexes[0].ProvisionedThroughput',
       code: 'UnexpectedParameter',
       time: Thu Feb 05 2015 16:30:05 GMT-0800 (PST) },
     { [UnexpectedParameter: Unexpected key 'LastDecreaseDateTime' found in params.ProvisionedThroughput]
       message: 'Unexpected key \'LastDecreaseDateTime\' found in params.ProvisionedThroughput',
       code: 'UnexpectedParameter',
       time: Thu Feb 05 2015 16:30:05 GMT-0800 (PST) },
     { [UnexpectedParameter: Unexpected key 'LastIncreaseDateTime' found in params.ProvisionedThroughput]
       message: 'Unexpected key \'LastIncreaseDateTime\' found in params.ProvisionedThroughput',
       code: 'UnexpectedParameter',
       time: Thu Feb 05 2015 16:30:05 GMT-0800 (PST) } ],
  time: Thu Feb 05 2015 16:30:05 GMT-0800 (PST) }
{ [Error: write EPIPE] code: 'EPIPE', errno: 'EPIPE', syscall: 'write' }
```

**Comment by mick** [comment]
@sbma44 unrelated, but fixed in 0.8.1
